src/devices/src/virtio/mod.rs

```diff
@@ -49,6 +51,7 @@ mod device_status {
 /// Type 0 is not used by virtio. Use it as wildcard for non-virtio devices
 pub const TYPE_NET: u32 = 1;
 pub const TYPE_BLOCK: u32 = 2;
+pub const TYPE_RNG: u32 = 4;
 pub const TYPE_BALLOON: u32 = 5;
```

Comment:
We can use [the virtio bindings from rust-vmm](https://github.com/rust-vmm/vm-virtio/tree/main/crates/virtio-bindings) e.g. [`virtio_bindings::VIRTIO_ID_RNG`](https://docs.rs/virtio-bindings/0.1.0/virtio_bindings/bindings/virtio_blk/constant.VIRTIO_ID_RNG.html) etc.

Reply:
I was wondering why don't we use this, but I see the project is archived: https://github.com/rust-vmm/virtio-bindings

Reply:
https://github.com/rust-vmm/virtio-bindings are the old bindings, what I linked https://github.com/rust-vmm/vm-virtio/tree/main/crates/virtio-bindings are the new bindings which are not archived.

Reply:
I'm okay with it not being a part of this PR, in this case I would think it useful to create an issue on transiting to using the rust-vmm bindings instead of our own.

Reply:
Before opening an issue, shall we try to figure out whether there is any issue with versioning?

Reply:
I think questions/concerns like this might reasonably be discussed under such an issue.